ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -269,25 +393,59 @@ type opennebula_oned = {
     "datastore_mad" : opennebula_datastore_mad
     "hm_mad" : opennebula_hm_mad
     "auth_mad" : opennebula_auth_mad
+    "market_mad" : opennebula_market_mad
     "default_cost" : opennebula_default_cost
+    "listen_address" : type_ipv4 = '0.0.0.0'
+    "vnc_ports" : opennebula_vnc_ports
+    "vlan_ids" : opennebula_vlan_ids
+    "vxlan_ids" : opennebula_vxlan_ids
     "tm_mad_conf" : opennebula_tm_mad_conf[] = list(
-        dict(), 
+        dict("ds_migrate", true), 
         dict("name", "lvm", "clone_target", "SELF"), 
-        dict("name", "shared"), 
+        dict("name", "shared", "ds_migrate", true), 
         dict("name", "fs_lvm", "ln_target", "SYSTEM"), 
         dict("name", "qcow2"), 
-        dict("name", "ssh", "ln_target", "SYSTEM", "shared", false), 
+        dict("name", "ssh", "ln_target", "SYSTEM", "shared", false, "ds_migrate", true), 
         dict("name", "vmfs"), 
-        dict("name", "ceph", "clone_target", "SELF")
+        dict("name", "ceph", "clone_target", "SELF", "ds_migrate", false),
+        dict("name", "iscsi_libvirt", "clone_target", "SELF", "ds_migrate", false),
+        dict("name", "dev", "clone_target", "NONE"),
+        dict("name", "vcenter", "clone_target", "NONE"),
+    )
+    "ds_mad_conf" : opennebula_ds_mad_conf[] = list(
+        dict(),
+        dict("name", "ceph", "required_attrs", "DISK_TYPE,BRIDGE_LIST,CEPH_HOST,CEPH_USER,CEPH_SECRET",
+             "marketplace_actions", "export"),
+        dict("name", "dev", "required_attrs", "DISK_TYPE",
+             "persistent_only", true),
+        dict("name", "iscsi_libvirt", "required_attrs", "DISK_TYPE,ISCSI_HOST",
+             "persistent_only", true),
+        dict("name", "fs", "marketplace_actions", "export"),
+        dict("name", "lvm", "required_attrs", "DISK_TYPE,BRIDGE_LIST"),
+        dict("name", "vcenter", "required_attrs", "VCENTER_CLUSTER", "persistent_only", true,
+             "marketplace_actions", "export"),
+    )
+    "market_mad_conf" : opennebula_market_mad_conf[] = list(
+        dict("public", true),
+        dict("name", "http", "required_attrs", "BASE_URL,PUBLIC_DIR", "app_actions", "create, delete, monitor"),
```

Comment:
use a `list("create", "delete", "monitor")` instead of `"create, delete, monitor"`, same for line below